specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/scheduledActions.json

```diff
@@ -0,0 +1,940 @@
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The details of the error.",
+          "$ref": "#/definitions/ErrorDetails"
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "The details of the error.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "FileDestination": {
+      "type": "object",
+      "description": "Destination of the view data. This is optional. Currently only csv format is supported.",
+      "properties": {
+        "fileFormats": {
+          "type": "array",
+          "description": "Destination of the view data. Currently only csv format is supported.",
+          "items": {
+            "$ref": "#/definitions/FileFormat"
+          }
+        }
+      }
+    },
+    "FileFormat": {
+      "type": "string",
+      "description": "Destination of the view data. Currently only csv format is supported.",
+      "enum": [
+        "Csv"
+      ],
+      "x-ms-enum": {
+        "name": "FileFormat",
+        "modelAsString": true
+      }
+    },
+    "Language": {
+      "type": "string",
+      "description": "Language in which the email is localized, for ex: 'Fr'. When the specified language is not supported, it will be defaulted to english. The supported languages are cs (Czech), da (Danish), de (German), en (English), es (Spanish), fr (French), hu (Hungarian), id (Indonesian), it (Lithuanian), ja (Japanese), ko (Korean), nb (Norwegian Bokmål), nl (Dutch), pl (Polish), pt (Portuguese), ru (Russian), sv (Swedish), tr (Turkish), zh (Chinese) "
+    },
+    "NotificationEmail": {
+      "type": "string",
+      "description": "Email address of the point of contact that should get the unsubscribe requests and notification emails."
+    },
+    "NotificationProperties": {
+      "type": "object",
+      "description": "The properties of the scheduled action notification.",
+      "required": [
+        "to",
+        "subject"
+      ],
+      "properties": {
+        "to": {
+          "description": "Array of email addresses.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "pattern": "^[A-Za-z0-9._%+-]+@(?:[A-Za-z0-9-]+\\.)+[A-Za-z]{2,}$"
+          },
+          "minItems": 1,
+          "maxItems": 20
+        },
+        "language": {
+          "description": "Locale of the email.",
+          "$ref": "#/definitions/Language"
+        },
+        "message": {
+          "description": "Optional message to be added in the email. Length is limited to 250 characters.",
+          "type": "string"
+        },
+        "regionalFormat": {
+          "description": "Regional format of the email.",
```

Comment:
Have you talked about PII, privacy review, avoiding spam complaints by having easy way to unsubscribe or opt out, etc?

Reply:
Yes, we did privacy review and the email describes the way to unsubscribe.